modules/javafx.base/src/main/java/com/sun/javafx/collections/VetoableListDecorator.java

```diff
@@ -110,6 +112,7 @@ public boolean setAll(E... elements) {
 
     @Override
     public boolean setAll(Collection<? extends E> col) {
+        Objects.requireNonNull(col);
         onProposedChange(Collections.unmodifiableList(new ArrayList<>(col)), 0, size());
```

Comment:
It's good to make it explicit.  It already threw NPE because of `new ArrayList<>(col)`, so this is not a change.